### PR TITLE
add smoke test label to e2e suite

### PR DESCRIPTION
**Pull request by mozrokafor**
# References: 
Jira: [MNTOR-2108](https://mozilla-hub.atlassian.net/browse/MNTOR-2108)

# Description
In order to concentrate on testing only vital flows during PR pushes

### CODE REVIEW

File: src/e2e/specs/auth.spec.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import { test, expect } from '../fixtures/basePage.js'
 
-test.describe('Authentication flow verification', () => {
+test.describe('Authentication flow verification @smoke', () => {
```
[the remark this answers is not in the document]
Comment (mozrokafor) [reply src/e2e/specs/auth.spec.js:7]: yes some aspects of the landing page, will be included, this is just to setup the process. I'm going to add back the necessary tests in follow up tickets

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -73,7 +72,7 @@ jobs:
         run: npx playwright install --with-deps
 
       - name: Run Playwright tests
-        run: npm run e2e
+        run: npm run e2e:smoke
```
[the remark this answers is not in the document]
Comment (mozrokafor) [reply .github/workflows/playwright.yml:75]: all e2e tests while important and is quick feedback does not need to run during PR, eventually we'll reach a point of the tests either running too slow and other maintenance issues. This allows the team to keep running only vital flows during the PR workflow

[the remark this answers is not in the document]
Comment (mansaj) [reply .github/workflows/playwright.yml:75]: Makes sense, though I would say we'd want to run the entire suite at some point of the CICD pipeline. Perhaps when merging from one environment to another (dev to stage)

[the remark this answers is not in the document]
Comment (mansaj) [reply .github/workflows/playwright.yml:75]: Otherwise, why would we bother writing anything other than smoke tests

**Review by mansaj — APPROVED** [review]

**Pull request merged**